### PR TITLE
[BTAT-9826] - Added penalties tile and controller logic

### PR DESCRIPTION
**Pull request by BackRowOla**
Screenshot 2021-11-09 at 16 00 31

app-config-base PR:  https://github.com/hmrc/app-config-base/pull/6388

### CODE REVIEW

**Comment by BackRowOla** [comment]
app-config-staging PR: https://github.com/hmrc/app-config-staging/pull/9436

File: test/controllers/VatDetailsControllerSpec.scala [hunk 1]
```diff
@@ -486,13 +531,33 @@ class VatDetailsControllerSpec extends ControllerBaseSpec {
...
+            Right(Some(obligations)),
+            Right(Some(payments)),
+            Right(customerInformationMax),
+            Some(penaltiesSummaryModel)
+          )
+        }
+        result shouldBe expectedContent
+      }
+    }
   }
```
Comment (daparkes) [inline test/controllers/VatDetailsControllerSpec.scala:561]: Could you add a test in here for if the user has no penalties

File: app/testOnly/controllers/PenaltiesController.scala [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+
+import javax.inject.Inject
+
+class PenaltiesController @Inject()(penaltiesHomeView: PenaltiesHomeView,
+                                    mcc: MessagesControllerComponents,
+                                    implicit val appConfig: AppConfig) extends FrontendController(mcc) with I18nSupport{
+
+  def show(): Action[AnyContent] = Action { implicit request =>
+    println("[Pen Connector][Show] - in the show action!!!!")
```
Comment (daparkes) [inline app/testOnly/controllers/PenaltiesController.scala:32]: Did you mean to leave this in?

**Review by daparkes — CHANGES_REQUESTED** [review]

File: app/testOnly/controllers/PenaltiesController.scala (hunk 2 above)
Comment (BackRowOla) [reply app/testOnly/controllers/PenaltiesController.scala:32]: No i didn't 😅

**Review by daparkes — CHANGES_REQUESTED** [review]
Could you also add the Welsh for the tile? It's in the spreadsheet here https://jira.tools.tax.service.gov.uk/browse/BTAT-9793

**Review by daparkes — APPROVED** [review]

**Pull request merged**